packages/interaction-kit/src/interactions/application-commands/context-menu-interaction.ts

```diff
@@ -53,7 +53,7 @@ export default class ContextMenuInteraction<
 			case ApplicationCommandType.USER:
 				// @ts-expect-error This is set at runtime but it's guaranteed to be the generic
 				this.commandType = ApplicationCommandType.USER;
-				// @ts-expect-error Same as above
+				// @ ts-expect-error Same as above
```

Comment:
Not sure if it works with the space, I would leave it as is.
```suggestion
				// @ts-expect-error Same as above
```

Reply:
Idk it yelled at me when editing